toggl-open-source/toggldesktop
# Update Windows in-app notification style to reflect the new design

### 📒 Description
Update Windows in-app notification style to reflect the new design

### 🕶️ Types of changes
 - **Improvement** (non-breaking change which fixes an issue)

### 🤯 List of changes
- [x] Remove in-app message button text capitalization
- [x] Update InAppNotification's button styles

### 👫 Relationships
Closes #4314 

### 🔎 Review hints
Testing
1. Check out `rebranding/upcoming-rebranding-notification` branch (cf8104b827d7e276c6527a7e9b9f416a8c5a0d69) (contains the current branch + commits to help testing the future notification message).
2. Comment out these lines: https://github.com/toggl-open-source/toggldesktop/pull/4316/files#diff-9d1119db15cbea0e1551e5c61ddde603R1566-R1569
3. EITHER change the date on your computer to 1st September 2020 because the message should be shown since 31st August, or change OR  comment out the code that checks the 'from' date.
4. Rebuild the solution.
5. Run the app, observe the notification.
